Cabal/src/Distribution/Simple/Setup/Haddock.hs

```diff
@@ -442,7 +442,7 @@ defaultHaddockProjectFlags =
 haddockProjectCommand :: CommandUI HaddockProjectFlags
 haddockProjectCommand =
   CommandUI
-    { commandName = "v2-haddock-project"
+    { commandName = "haddock-project"
```

Comment:
I think CI fails because of this change. Basically, `v2-` has semantic meaning in the `Cabal` library.